scripts/qa_crowbarsetup.sh

```diff
@@ -2536,6 +2537,16 @@ function oncontroller_testsetup()
     . .openrc
 
     export LC_ALL=C
+
+    # prepare test image with the -test packages containing functional tests
+    if iscloudver 6plus; then
+        local mount_dir="/tmp/Cloud-Testing"
+        rsync_iso "$CLOUDSLE12DISTPATH" "$CLOUDSLE12TESTISO" "$mount_dir"
+        zypper -n ar --refresh -f "$mount_dir" cloud-test
+
+        ensure_packages_installed python-novaclient-test
```

Comment:
We start with novaclient because novaclient's Kilo version has already functional tests. For other clients, functional tests are only in the Liberty. We need a) move to Liberty and b) rework some packages to include the functional tests into the -test package.